tests/unit/versioning/VersionID.js

```diff
@@ -0,0 +1,50 @@
+const VID = require('../../../lib/versioning/VersionID.js');
+const assert = require('assert');
+
+function randkey(length = 15) {
```

Comment:
Ah, about the answer, it's there to represent the approximated length of a versionId string. I will move it to the caller.